classes/Http/Form/SignupForm.php

```diff
@@ -35,44 +37,31 @@ class SignupForm extends Form
         'url',
     ];
 
-    /**
-     * Validate all methods by calling all our validation methods
-     *
-     * @param string $action
-     *
-     * @return bool
-     */
-    public function validateAll($action = 'create')
+    public function validateAll(string $action = 'create'): bool
     {
         $this->sanitize();
+
         $validPasswords = true;
         $agreeCoc       = true;
 
-        if ($action == 'create') {
+        if ($action === 'create') {
             $validPasswords = $this->validatePasswords();
             $agreeCoc       = $this->validateAgreeCoc();
         }
 
-        $validEmail        = $this->validateEmail();
-        $validFirstName    = $this->validateFirstName();
-        $validLastName     = $this->validateLastName();
-        $validCompany      = $this->validateCompany();
-        $validTwitter      = $this->validateTwitter();
-        $validUrl          = $this->validateUrl();
-        $validSpeakerPhoto = $this->validateSpeakerPhoto();
-        $validSpeakerInfo  = true;
-        $validSpeakerBio   = true;
+        $validSpeakerInfo = true;
 
         if (!empty($this->taintedData['speaker_info'])) {
             $validSpeakerInfo = $this->validateSpeakerInfo();
```

Comment:
What if we move the check to see if it is empty to the ```validateSpeakerInfo()``` function?
And the same for the speaker bio?

Reply:
👍 for cleaning up further!